src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -3387,4 +3387,70 @@
       </Comments>
     </Property>
   </Node>
+  <Node Name="ICollectionExpressionOperation" Base="IOperation"  HasType="true">
+    <Comments>
+      <summary>
+        Represents a collection expression.
+        <para>
+          Current usage:
+          (1) C# collection expression.
+        </para>
+      </summary>
+    </Comments>
+    <Property Name="ConstructMethod" Type="IMethodSymbol?">
+      <Comments>
+        <summary>
+          Method used to construct the collection.
+          <para>
+            If the collection type is an array, span, array interface, or type parameter, the method is null;
+            if the collection type has a [CollectionBuilder] attribute, the method is the builder method;
+            otherwise, the method is the collection type constructor.
+          </para>
+        </summary>
+      </Comments>
+    </Property>
+    <Property Name="Elements" Type="ImmutableArray&lt;IOperation&gt;">
+      <Comments>
+        <summary>
+          Collection expression elements.
+          <para>
+            If the element is an expression, the entry is the expression, with a conversion to
+            the target element type if necessary;
+            otherwise, the entry is an ISpreadOperation.
+          </para>
+        </summary>
+      </Comments>
+    </Property>
+  </Node>
+  <Node Name="ISpreadOperation" Base="IOperation" HasType="true">
```

Comment:
>HasType="true" [](http://example.com/codeflow?start=50&length=14)

That seems unexpected. What's the type of a spread?

Reply:
Good catch, thanks.